### DISCUSSION TITLE
1.88 dont work on real android device

### DISCUSSION THREAD

**Discussion by vovan4ik123-engine** [ocornut/imgui#5578] [New Users Build/Link/Run issues ONLY!]
ImGUI 1.88 dont work on real android device.

All GUI elements (buttons, sliders, check boxes,...) does not react on screen touches.
GuiDemo from library is not clickable.

Tried on Samsung A51 (Android 12).
Library without my changes.
Here is my Cmake for build:

cmake_minimum_required(VERSION 3.10)

add_library(ImGUI-static STATIC imgui.cpp
                                imgui_demo.cpp
                                imgui_draw.cpp
                                imgui_tables.cpp
                                imgui_widgets.cpp
                                backends/imgui_impl_sdl.cpp
                                backends/imgui_impl_opengl3.cpp)

target_include_directories(ImGUI-static PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ../SDL2/include)

**Comment by rokups** [comment]
You did not describe issue you are having.
